src/index.js

```diff
@@ -76,22 +79,30 @@ exports.ensureAuthenticated = (request, response, next) => {
 
 let app = null;
 
-function buildSchema(Implementation) {
-  const { opts } = Implementation;
-
-  const absModelDirs = opts.modelsDir ? path.resolve('.', opts.modelsDir) : undefined;
-  return requireAllModels(Implementation, absModelDirs)
-    .then((models) => {
-      const integrator = new Integrator(opts, Implementation);
-      return Schemas.perform(Implementation, integrator, models, opts)
-        .thenReturn([models, integrator]);
+function buildSchema() {
+  const { lianaOptions, Implementation } = configStore;
+  const absModelDirs = configStore.modelsDir ? path.resolve('.', configStore.modelsDir) : undefined;
+  return requireAllModels(absModelDirs)
+    .then(async (models) => {
+      configStore.integrator = new Integrator(lianaOptions, Implementation);
+      await Schemas.perform(
+        Implementation,
+        configStore.integrator,
+        models,
+        lianaOptions,
+      );
+      return models;
     });
 }
 
 exports.init = (Implementation) => {
   const { opts } = Implementation;
+
+  configStore.Implementation = Implementation;
+  configStore.lianaOptions = opts;
+
   if (opts.onlyCrudModule === true) {
```

Comment:
[Misc] What is the `onlyCrudModule` exactly?

Reply:
It was used by lumber-graphql to generate the crud routes but without forestadmin other routes. Should be removed in another PR